app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -159,7 +173,10 @@ class SubscriptionController @Inject()
             addressLookUpConnector.getAddressDetails(id).flatMap { address =>
               desAddressForm.bindAddressLookupFrontendAddress(details.utr, address).fold(
                 formWithErrors => Future successful Ok(html.address_form_with_errors(formWithErrors)),
-                validDesAddress => subscribe(details, validDesAddress).map(redirectSubscriptionResponse)
+                validDesAddress => {
+                  mark("Count-Subscription-AddressLookup-Success")
+                  subscribe(details, validDesAddress).map(redirectSubscriptionResponse)
+                }
               )
             }
           }.getOrElse(Future.successful(sessionMissingRedirect()))
```

Comment:
mark() ?

Reply:
as above, we aim to monitor only key points